src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -447,14 +447,17 @@ private void updateSupplementaryEvidence(
             callBackEvent.exceptionRecordId
         );
 
+        CaseDetails targetCase = ccdApi.getCase(targetCaseCcdRef, callBackEvent.exceptionRecordJurisdiction);
```

Comment:
if this call fails will the caseworker shown an error screen. If this call fails they should be shown with a message to retry.

Reply:
Also test case is missing when this call throws an exception

Reply:
I'll update the pr

Reply:
When the ccd call fails, the error will be displayed to the caseworker. see https://github.com/hmcts/bulk-scan-orchestrator/blob/a0b8a51ac54e9a57863b7ca7c5c3483596a90bee/src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java#L235-L248

I've updated the integration tests for invalid case reference and case not found scenarios.